docs/inspections.md

```diff
@@ -26,6 +26,16 @@ items within a box can be inspected. If `within_box_proportion < 1`, only the
 first `n = within_box_proportion * items_per_box` items in each box will be
 inspected.
 
+## Tolerance level
+The contamination tolerance level can be determined by:
+
+```
+inspection:
+  tolerance_level: 0
+```
+
+The simulation provides a count of the number of missed consigments with contamination rates below the specified `tolerance_level`, which can be any value between 0 and 1. The resulting count can be used to adjust the overall missed contaminated consignments when evaluating inspection efficacy. For example, if an inspection strategy fails to detect 100 out of 500 contaminated consignments (20% slippage), but 80 of the missed consignments had a `contamination_rate` < `tolerance_level`, you may choose to adjust the inspection slippage rate to not include the 80 consignments above the tolerance level (decreases to 4% slippage).
```

Comment:
"to not include" sounds like it should be "to include only" For me, it fits with the condition here and in the code.

Reply:
Yes, I mixed that up. Let me know if this change makes more sense and if this PR can be merged. Thanks!

Reply:
I think the user expectation is that you get the adjusted slippage (as well or only), so that might be the origin of our confusion. However, we can add the adjusted slippage rate later. (I think it would be quite questionable to report only the adjusted one.)